### PR TITLE
upgrade to subgraph v39_3

### PR DESCRIPTION
**Pull request by dkent600**
Resolves: https://github.com/daostack/alchemy/issues/1589

### CODE REVIEW

File: src/components/Daos/DaosPage.tsx [hunk 1]
```diff
@@ -98,7 +98,8 @@ class DaosPage extends React.Component<IProps, IState> {
     // Always show Genesis Alpha first
     let finalDAOList = allDAOs.filter((d: DAO) => d.staticState.name === "Genesis Alpha" && d.staticState.name.toLowerCase().includes(search));
 
-    if (process.env.NODE_ENV === "staging") {
+     // eslint-disable-next-line no-extra-boolean-cast
+    if (process.env.SHOW_ALL_DAOS === "true") {
```
Comment (dkent600) [inline src/components/Daos/DaosPage.tsx:102]: This should be removed.  It doesn't work and is done correctly in another PR

Comment (orenyodfat) [reply src/components/Daos/DaosPage.tsx]: it is doing what it suppose to do .

Comment (dkent600) [reply src/components/Daos/DaosPage.tsx]: but it will always return false.

Comment (dkent600) [reply src/components/Daos/DaosPage.tsx]: plus everything else I pointed out

Comment (orenyodfat) [reply src/components/Daos/DaosPage.tsx]: NO . it is return TRUE!
and it is working
